<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payment</class>
 <widget class="QDialog" name="payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>결제</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>카드결제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="point_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>사용</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>카드회사</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="card_txt">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>입력</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>카드번호</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cardnum_txt">
      <property name="placeholderText">
       <string>입력</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>포인트</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="point_txt">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>할인금액</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="sale_txt"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>결제금액</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="total_txt">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>선택안함</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>일반자차</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>완전자차</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="insurance_txt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pay_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>결제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
